app/src/util/form-schema/index.ts

```diff
@@ -57,29 +57,33 @@ export interface SuggestedActivity {
 }
 
 export interface Methodology {
-  id: string;
-  disabled?: boolean;
-  activities?: Activity[];
-  inputRequired?: string[];
-  formula?: string;
-  fields?: any[];
-  suggestedActivities?: SuggestedActivity[];
-  suggestedActivitiesId?: string;
+    id: string;
```

Comment:
Why all of these formatting changes? Please configure your editor for 2 spaces and enable format on save for prettier.